src/test/java/hw9/BoardTest.java

```diff
@@ -0,0 +1,112 @@
+import org.testng.annotations.Test;
+import utils.Endpoints;
+
+
+public class BoardTest extends BasicTest {
+
+    static String boardName = RandomStringUtils.random(40, true, true);
+
+    static String listName = RandomStringUtils.random(40, true, true);
+
+    @Test
+    public static void createBoard() {
+        board = given().spec(baseRequestSpecification)
+                .when()
+                .queryParam("name", boardName)
+                .basePath(Endpoints.Board_Url)
+                .post()
+                .then()
+                .body("name", equalTo(boardName))
+                .spec(responseSpecification).log().all()
+                .extract().body().as(BoardEntity.class);
+    }
+
+    @Test
+    public void getInfoAboutBoard() {
+        if (board == null) {
+            createBoard();
+        }
+        board = given().spec(baseRequestSpecification)
+                .when()
+                .basePath(Endpoints.Board_ID_Url)
+                .pathParam("id", board.id())
+                .get()
+                .then()
+                .spec(responseSpecification)
+                .body("name", equalTo(boardName))
+                .log().all()
+                .extract().body().as(BoardEntity.class);
+    }
+
+    @Test
+    public void editBoard() {
+        if (board == null) {
+            createBoard();
+        }
+        String descriptionForBoard = RandomStringUtils.random(100, true, true);
+        var boardNew = BoardEntity.builder().desc(descriptionForBoard).id(board.id()).build();
+        board = given().spec(baseRequestSpecification)
+                .when()
+                .basePath(Endpoints.Board_ID_Url)
+                .pathParam("id", board.id())
+                .body(boardNew)
+                .put()
+                .then()
+                .body("desc", equalTo(descriptionForBoard))
+                .body("name", equalTo(boardName))
+                .spec(responseSpecification)
+                .log().all()
+                .extract().body().as(BoardEntity.class);
+    }
+
+    @Test
+    public static void deleteBoard() {
+        if (board != null) {
+            given()
+                    .spec(baseRequestSpecification)
+                    .when()
+                    .basePath(Endpoints.Board_ID_Url)
+                    .pathParam("id", board.id())
+                    .delete()
+                    .then().spec(responseSpecification);
+            given()
+                    .spec(baseRequestSpecification)
+                    .when()
+                    .basePath(Endpoints.Board_ID_Url)
+                    .pathParam("id", board.id())
+                    .get()
+                    .then()
+                    .spec(notFoundResponseSpecification);
+            System.out.printf("Board with id = %s deleted%n", board.id());
+            board = null;
+        } else {
+            System.out.println("Board is not found. May be board was removed before.");
+        }
```

Comment:
That means we really don't cover this scenario and can't be assured that this functionality works as expected. More than that we can be facing false positive run.